docs/source/integrating/version-5-auth.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Social Sign-in & Email Auth
+permalink: /v5/integration/auth/
+---
+
+## Authentication Strategies 
+
+There are several strategies to choose from when it comes to user authentication. Login methods can be enabled or disabled for both the **Coral Admin** and the **Comment Stream** separately thus allowing you to use different methods for different user groups. 
+
+You can also use "Allow Registration" to restrict or allow users that have not signed up before to register and create new user accounts. 
+
+To configure authentication for your instance of Coral, go to **Configure** > **Authentication**. 
+
+## Login with email authentication
+Best when you want Coral to manage all user registration, creation and authentication. Requires the least amount of setup and configuration. Users register by creating a Username (must be unique across users, no spaces or special chars), Email, and Password. All user details are stored in Coral's database. 
+
+By Default - users are sent a verification email and are prompted to verify email addresss, however completing the email verification step is not required to submit comments. 
+
+
+## Login with OpenID Connect
+
+Allows integration with your existing OpenID Connect authentication provider. When enabled this strategy adds a "Login with ___", button to Coral's login modal with the value you've specified for `Provider Name`. 
+
+Authentication is handled by passing an access token parameter to the comment stream embed when placing the call to render it. To learn more please visit: https://openid.net/connect/
```

Comment:
Enabling this option will simply enable the Login With button, and not necessarily the token passing method as described here.

Reply:
I wasn't trying to imply that enabling it would enable the token functionality, but that tokens are how it works once it's been enabled. Will push new commit.
Rephrasing to: 
> With OpenID Connect authentication is handled by...